### PR TITLE
LPS-116767 Update liferay-ckeditor to v4.14.1-liferay.3

### PR DESCRIPTION
**Pull request by carloslancha**
Updating liferay-ckeditor to 4.14.1-liferay.3

This will fix https://issues.liferay.com/browse/LPS-116767

Steps to test:
- Go to Content & Data > Web Content
- Add New Basic Web Content

### CODE REVIEW

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"dependencies": {
 		"ckeditor4-react": "1.0.1",
 		"frontend-js-web": "*",
-		"liferay-ckeditor": "4.14.1-liferay.2"
+		"liferay-ckeditor": "^4.14.1-liferay.3"
```
Comment (wincent) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/package.json:5]: Let's keep this exactly locked like we used to.

Comment (carloslancha) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/package.json:5]: Ups, I did `yarn add` and didn't notice it

**Comment by carloslancha** [comment]
ci:stop

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:forward

**Pull request closed without merging**